src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,82 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static StorageDaoImpl storage;
+    private static RegistrationServiceImpl service;
+
+    @BeforeEach
+    public void setUp() {
+        storage = new StorageDaoImpl();
+        service = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void register_normalUser_Ok() {
+        User normalUser = new User("MisterSandman", "BringMeADreams", 20);
+        User user = service.register(normalUser);
+        assertNotNull(user);
+        assertEquals(normalUser, user);
+    }
+
+    @Test
+    public void register_existingUser_NotOk() {
+        User bob = new User("bobTheOne", "iAmHereAlice", 23);
+        User theSameBob = new User("bobTheOne", "iAmHereAlice", 23);
+        storage.add(bob);
+        storage.add(theSameBob);
+        assertThrows(RuntimeException.class, () -> {
+            service.register(new User("bobTheOne", "iAmHereAlice", 23));
+        });

```

Comment:
```suggestion
    public void register_existingUser_notOk() {
        User bob = new User("bobTheOne", "iAmHereAlice", 23);
        storage.add(bob);
        assertThrows(RuntimeException.class, () -> 
            service.register(new User("bobTheOne", "iAmHereAlice", 23))
        );

```
в Storage не повинно бути двох однакових користувачів і для тестування це непотрібно. 
кожна частина назви тестового методу, повинна починатися з малої літери (перевір це всюди).